__tests__/app.test.js

```diff
@@ -8,6 +8,8 @@ const {
   commentData,
 } = require("../db/data/test-data/index");
 const seed = require("../db/seeds/seed");
+const { readFile } = require("fs/promises");

```

Comment:
You dont have to actually readFile a JSON file, you can just require it directly like
`const endpoints = require('../..etc')`